compressor/base.py

```diff
@@ -209,20 +210,21 @@ def hunks(self, forced=False):
                 precompiled, value = self.precompile(value, **options)
 
             if enabled:
-                yield self.filter(value, **options)
+                yield self.filter(value, self.cached_filters, **options)
+            elif precompiled:
+                if CssAbsoluteFilter in self.cached_filters:
```

Comment:
Add a small comment explaining that/why we want to apply `CssAbsoluteFilter` even if we're disabled